jose.py

```diff
@@ -209,13 +209,14 @@ def decrypt(jwe, jwk, adata='', validate_claims=True, expiry_seconds=None):
     # decrypt body
     ((_, decipher), _), ((hash_fn, _), mod) = JWA[header['enc']]
 
-    if header.get(_TEMP_VER_KEY) == _TEMP_VER:
+    version = header.get(_TEMP_VER_KEY, None)
```

Comment:
I usually like to be explicit. It allows less room for confusion and misunderstandings. But I can address that if needed.